Rubberduck.Parsing/Grammar/VBAParser.g4

```diff
@@ -877,7 +878,8 @@ commentOrAnnotation :
 ;
 remComment : REM whiteSpace? commentBody;
 comment : SINGLEQUOTE commentBody;
-commentBody : (LINE_CONTINUATION | ~NEWLINE)*;
+// commentBody must terminate with a NEWLINE, see VBA Language spec, section 3.3.1 and 3789
```

Comment:
The comment is not exactly true. In essence, the specification states that a comment has to end with the end of a logical line, which includes the end of the file.